lib/faker/geohash.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Faker.Geohash do
+    import Faker.Address, only: [latitude: 0, longitude: 0]
+    alias Faker.Address
+
+    @moduledoc """
+    Function for generate geohashes.
+    """
+    @doc """
+    Return random geohash.
+    """
+    @spec geohash() :: String.t
+    def geohash do
+      import Geohash, only: [encode: 2]
```

Comment:
You don't need to `import` in case of a fully qualified call to a function. Also, the arity of imported function doesn't match function on next line.